browser/src/NeovimInstance.ts

```diff
@@ -126,6 +126,14 @@ export class NeovimInstance extends EventEmitter implements INeovimInstance {
                             this.emit("event", eventName, eventContext)
                         } else if (pluginMethod === "window_display_update") {
                             this.emit("window-display-update", args[0][0], args[0][1])
+                        } else if (pluginMethod === "api_info") {
+                            const apiVersion = args[0][0]
+                            if (apiVersion.minor < 1) {
```

Comment:
… also, thinking about it, shouldn't this be able to handle the case the Neovim API hits a major version > 0? I.e.
```javascript
if (apiVersion.major < 1 && apiVersion.minor < 2 && apiVersion.patch < 6) {
  // handle incompatible version
}